src/pom.xml

```diff
@@ -321,6 +331,55 @@
     </profiles>
 
     <build>
+        <pluginManagement>
+            <plugins>
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-clean-plugin</artifactId>
+                    <version>3.0.0</version>
```

Comment:
Could all the versions in the `pluginManagement` section also be extracted to properties?

Reply:
Done in 2dd454f.